### ISSUE TITLE
Does onnx support momentum=None in nn.BatchNorm?

### ISSUE THREAD

**Issue by panjiangwei** [pytorch/pytorch#40074]
When momentum=None, it's supposed to be simple average in PyTorch.

When I set verbose=True in torch.onnx.export, I see that momentum is set to 1.0 in the onnx batch_norm op.  Is this a bug or expected?

cc @houseroad @spandantiwari @lara-hdr @BowenBao @neginraoof

**Comment by spandantiwari** [comment]
@panjiangwei - which version of PyTorch are you using? Also, can you elaborate where and why there is a bug?

**Comment by panjiangwei** [comment]
I used PyTorch 1.5.0.  

In the documentation of BatchNorm1d, it says when momentum=None, the layer will perform simple average ([code](https://github.com/pytorch/pytorch/blob/eace0533985641d9c2f36e43e3de694aca886bd9/torch/nn/modules/batchnorm.py#L114-L115)).  When I run torch.onnx.export, I see that the momentum parameter of the onnx batch_norm op is set to 1.0.  I was wondering if this is intentional or a bug.  Thanks!

**Comment by spandantiwari** [comment]
@panjiangwei - I would say this is intentional. `torch.onnx.export` API exports batch norm in eval mode where the running stats are frozen. Also, note that ONNX `BatchNormalization` op is defined differently from PyTorch (`MomentumPT = 1 - MomentumONNX`). This is reflected in the exporter [here](https://github.com/pytorch/pytorch/blob/0ddaaf6a92b85bf9708bbc1715e82da3f601231d/torch/onnx/symbolic_opset9.py#L1115).

In eval mode the momentum in PyTorch is set to 0 (seen [here](https://github.com/pytorch/pytorch/blob/eace0533985641d9c2f36e43e3de694aca886bd9/torch/nn/modules/batchnorm.py#L106)), and consequently you see it set to 1.0 in the ONNX model.
